src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/SingleShot.cs

```diff
@@ -125,7 +146,15 @@ protected virtual void Dispose(bool disposing)
             if (_handle != IntPtr.Zero)
             {
                 NNStreamerError ret = NNStreamerError.None;
-                ret = Interop.SingleShot.CloseSingle(_handle);
+                try
+                {
+                    ret = Interop.SingleShot.CloseSingle(_handle);
+                }
+                catch (DllNotFoundException e)
```

Comment:
DllNotFound is already checked in CheckNNStreamerSupport()

Reply:
ACK